src/java/org/apache/cassandra/tools/nodetool/CompressionDictionary.java

```diff
@@ -119,4 +132,135 @@ private static void displayProgress(TrainingState trainingState, long startTime,
                        status, sampleCount, sampleSizeMB, elapsedSeconds);
         }
     }
+
+    @Command(name = "list", description = "List available dictionaries of specific keyspace and table.")
+    public static class List extends AbstractCommand
+    {
+        @Parameters(index = "0", description = "The keyspace name", arity = "1")
+        private String keyspace;
+
+        @Parameters(index = "1", description = "The table name", arity = "1")
+        private String table;
+
+        @Override
+        protected void execute(NodeProbe probe)
+        {
+            try
+            {
+                TableBuilder tableBuilder = new TableBuilder();
+                TabularData tabularData = probe.listCompressionDictionaries(keyspace, table);
+                java.util.List<String> indexNames = tabularData.getTabularType().getIndexNames();
+
+                java.util.List<String> columns = new ArrayList<>(indexNames);
+                columns.remove(3);  // ignore raw dict
+                tableBuilder.add(columns);
+
+                for (Object eachDict : tabularData.keySet())
+                {
+                    final java.util.List<?> dictRow = (java.util.List<?>) eachDict;
+
+                    java.util.List<String> rowValues = new ArrayList<>();
+
+                    for (int i = 0; i < dictRow.size(); i++)
+                    {
+                        if (i == 3) // ignore raw dict
+                            continue;
+
+                        rowValues.add(dictRow.get(i).toString());
+                    }
+                    tableBuilder.add(rowValues);
+                }
+
+                tableBuilder.printTo(probe.output().out);
+            }
+            catch (Exception e)
+            {
+                probe.output().err.printf("Failed to list dictionaries: %s%n", e.getMessage());
+                System.exit(1);
+            }
+        }
+    }
+
+    @Command(name = "export", description = "Export dictionary from Cassandra to local file.")
+    public static class Export extends AbstractCommand
```

Comment:
add optional parameter to store to a file (simple redirection to file in the console works for now but might be not appropriate especially when dictionary bytes string is way too big for screen to fit in).